components/expenses/PayExpenseModal.js

```diff
@@ -193,6 +194,15 @@ const PayExpenseModal = ({ onClose, onSubmit, expense, collective, error }) => {
         {error && (
           <MessageBox type="error" withIcon my={3}>
             {error}
+            {/** TODO: Add a proper ID/Type to detect this error */}
+            {error.startsWith('Not enough funds in your existing Paypal preapproval') && (
+              <StyledLink href={`https://opencollective.com/${collective.host.slug}/dashboard`}>
+                <FormattedMessage
+                  id="PayExpenseModal.RefillBallanceError"
```

Comment:
Minor mistake; typo :wink: 

```suggestion
                  id="PayExpenseModal.RefillBalanceError"
```

Reply:
Done